### PR TITLE
Steering group minutes

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2024/2024-09-09-steering.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+## Attending
+
+Present/Contributing: Graeme Stewart, Pere Mato, Torre Wenaus, Eduardo Rodrigues, Benedikt Hegner, Liz Sexton-Kennedy, Michel Jouvin
+
+Apologies: Andrea Valassi, Nicole Skidmore
+
+## HSF Email Lists
+
+The old coordination list, `hsf-coordination@googlegroups.com`, has been renamed to `hsf-coordination-archive@googlegroups.com`
```
Comment (eduardo-rodrigues) [inline organization/_posts/2024/2024-09-09-steering.md:14]:
```suggestion
The old coordination list, `hsf-coordination@googlegroups.com`, has been renamed to `hsf-coordination-archive@googlegroups.com`.
```

File: organization/_posts/2024/2024-09-09-steering.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+- Email to the old list name will now *bounce*
+    - Otherwise people think they have contacted the HSF, but in reality haven't
+    - Using a new list name allows access to the email archive, in case it is needed (ask Graeme for access)
+    - Using a google list to forward to other lists is not possible in the *gratis* version (also meaning we can't have a list that goes to SG + activity conveners, unfortunately)
```
Comment (eduardo-rodrigues) [inline organization/_posts/2024/2024-09-09-steering.md:19]:
```suggestion
    - Using a Google list to forward to other lists is not possible in the *gratis* version (also meaning we can't have a list that goes to SG + activity conveners, unfortunately)
```

File: organization/_posts/2024/2024-09-09-steering.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+General agreement that we should contribute to the process.
+
+Key areas seem to be: generators, simulation, reconstruction, analysis and training/careers.
+
+**We need a team to manage this process and organise the work with the conveners and the wider community.**
+
+Volunteers: Michel, Graeme (we need a few more)
+
+- [ ] We need to add a couple more SG members,
```
Comment (eduardo-rodrigues) [inline organization/_posts/2024/2024-09-09-steering.md:76]:
```suggestion
- [ ] We need to add a couple more SG members
```

File: organization/_posts/2024/2024-09-09-steering.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+## HSF and SFT reporting to WLCG/LHCC
+
+LHCC referees meeting for WLCG is tomorrow: Eduardo will represent the HSF.
+
+Slides:
+
+- [Common and Community Software](https://docs.google.com/presentation/d/1361TObGpuKHpOLXwaubsBNdJBYPG5aV_32eYblKjpvI/edit?usp=sharing)
+- [HSF/community Software Report](https://docs.google.com/presentation/d/130zUrEp5gDfKQL6ZIcts7e00qOI697Q8RbLlHuVno3Y/edit?usp=sharing)
+
+Draft for the RRB report will be circulated.
```
Comment (eduardo-rodrigues) [inline organization/_posts/2024/2024-09-09-steering.md:91]:
```suggestion
Draft for the RRB report will be circulated shortly.
```

**Review by eduardo-rodrigues — APPROVED** [review]
LGTM.

**Review by hegner — APPROVED** [review]

**Comment by graeme-a-stewart** [comment]
Thank you both for reviewing!

**Pull request merged**